test/unit/init/ClusterManager.test.ts

```diff
@@ -0,0 +1,35 @@
+import cluster from 'cluster';
+import { cpus } from 'os';
+import { ClusterManager } from '../../../src';
+
+jest.mock('os', (): any => ({
+  ...jest.requireActual('os'),
+  cpus: jest.fn().mockImplementation((): any => [{}, {}, {}, {}, {}, {}]),
+}));
+jest.mock('cluster', (): any => ({
+  ...jest.requireActual('cluster'),
```

Comment:
What breaks if you don't add the rest of the os/cluster imports here?

Reply:
for `os`:
```
 FAIL  test/unit/init/ClusterManager.test.ts
  ● Test suite failed to run
                                                                                                                                                                                                                                                                                                                  
    TypeError: os.networkInterfaces is not a function

      at getIfaceInfo (node_modules/cacheable-lookup/source/index.js:41:40)
      at new CacheableLookup (node_modules/cacheable-lookup/source/index.js:109:17)
      at Object.<anonymous> (node_modules/oidc-provider/lib/helpers/request.js:11:19)
      at Object.<anonymous> (node_modules/oidc-provider/lib/helpers/request_uri_cache.js:7:17)
```
`cluster` has been changed a few minutes ago

Reply:
It cascades into a new function each time I mock the previous one..
* networkInterfacers
* type
* require(...).platform is not a function
* ...

Reply:
I'm guessing jest or something else is using those. Just wanted to make sure these were needed so is fine to keep it.